docs/source/optimizations/cq/cq_compile_css.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "eb89a144",
```

Comment:
<p>Nits:</p><ul><li>Update cirq-superstaq, Service(), and css.Service(api_key="token") to be in-line code</li><li>"ie" should be "i.e."</li><li>add link to https://docs-superstaq.readthedocs.io/en/stable/get_started/basics/basics_css.html#Set-up-access-to-Superstaq's-API for guide to setting environment variables.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/527/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done</p>